Zappy_IA_Report/NN/Code/Tutorial3.py

```
@@ -0,0 +1,37 @@
+import matplotlib.pyplot as plt
+
+data = keras.datasets.fashion_mnist
+(train_images, train_labels), (test_images, test_labels) = data.load_data()
+
+class_names = ['T-shirts/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+train_images = train_images/255.0
+test_images = test_images/255.0
+
+model = keras.Sequential([
+    keras.layers.Flatten(input_shape=(28,28)), 
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)